docs/extending/richtext_block.md

```diff
@@ -1,108 +1,50 @@
-# Creating custom RichText blocks [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+# Create custom RichText block [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
-To create a RichText [custom Page block](../guide/page/create_custom_page_block.md), you need to define its layout, provide templates, add a subscriber and register it as a service.
+In addition to existing blocks that you can use when creating a Page, you can 
```

Comment:
Do you need to repeat the info from the main Page block page? Maybe instead link to that page and indicate that this is just one specific case of a block?